portal-ecs.yml

```diff
@@ -1049,7 +1078,7 @@ Resources:
       # to keep this value small
       - Key: deregistration_delay.timeout_seconds
         Value: '5'
-      VpcId: vpc-9c9714f9
+      VpcId: !Ref 'VpcId'
```

Comment:
`{ 'Fn::ImportValue': !Sub '${ClusterStackName}-ClusterVpcId' }` should be used here.